README.md

```diff
@@ -1 +1,27 @@
-# MicroverseStudent
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> "Hello Microverse" project
+
+
+## Built With
+
+- HTML/CSS
+
+
+## Getting Started
+
+**This is the first repository created for the Microverse**
+
+This file was made for review.
+
+
+## Author
+
+👤 **Daniela Reis**
+
+
```

Comment:
- [ ] Kindly add your social links (or what you have of them) to your `README` file (GitHub, LinkedIn, Twitter).